CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 - Mark exceptions not handled by the user as `handled: false` ([#1535](https://github.com/getsentry/sentry-dart/pull/1535))
   - This will affect your release health data, and is therefore considered a breaking change.
-
+- Refrain from overwriting the span status for unfinished spans ([#1577](https://github.com/getsentry/sentry-dart/pull/1577))
+  - Older self-hosted sentry instances will drop transactions containing unfinished spans.
```

Comment:
Can we link the min. version of the required self-hosted now.

Reply:
This change was part of relay PR [relay/#1690](https://github.com/getsentry/relay/pull/1690), which was released with [relay /22.12.0](https://github.com/getsentry/relay/releases/tag/22.12.0). I couldn't find the sentry server version from the change logs that is using this relay version.